### PR TITLE
Update calibrate api

### PR DESCRIPTION
**Pull request by Priscilla-AL**
# <Feature/Chore/Hotfix/Topic/Bug Title>

**_WHAT DOES THIS PR DO?_**
Updates  calibrate API 

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
https://airqoteam.atlassian.net/jira/software/projects/DP/boards/44?assignee=5d91eed256bb950dd6489270&selectedIssue=DP-62

**_WHAT IS THE LINK TO THE PR BRANCH_**
https://github.com/airqo-platform/AirQo-api/tree/update-calibrate-api

**_HOW DO I TEST OUT THIS PR?_**
cd AirQo-api/src/calibrate
python -m venv [DESTINATION_FOLDER]
 [DESTINATION_FOLDER]/Scripts/activate
pip install -r requirements.txt

python jobs\regression.py  (one time)
flask run
NB: Make sure you add a GCP service account key under jobs/

SAMPLE REQUEST BODY
{ 
"datetime": "2020-07-15 13:00:00",
 "raw_values":[
          {
              "device_id":"aq_01",
               "sensor1_pm2.5": 44.12 , 
               "sensor1_pm10":54.20,
               "sensor2_pm2.5": 44.12 , 
               "sensor2_pm10":54.20,            
               "temperature":25.3, 
               "humidity":62.0
         }, 
         { 
                "device_id":"aq_02",
                "sensor1_pm2.5": 44.12 , 
               "sensor1_pm10":54.20,
               "sensor2_pm2.5": 44.12 , 
               "sensor2_pm10":54.20,          
                "temperature":26.5, 
                "humidity":57.0 
          }, 
          { 
                  "device_id":"aq_03",
                  "sensor1_pm2.5": 44.12 , 
                  "sensor1_pm10":54.20,
                 "sensor2_pm2.5": 44.12 , 
                 "sensor2_pm10":54.20,  
                  "temperature":27.0, 
                  "humidity":56.0 
            } 
                        ]
 }

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
[POST] http://127.0.0.1:4001/api/v1/calibrate 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
NA

**_ARE THERE ANY RELATED PRs?_**
NA

### CODE REVIEW

**Review by lilly-muyama — APPROVED** [review]
LGTM

**Review by NoahNsimbe — APPROVED** [review]
LGTM!

File: src/calibrate/requirements.txt [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 confluent-avro
 kafka-python
-python-dotenv
\ No newline at end of file
+python-dotenv
+gcsfs
+google-api-core
+google-auth
+google-auth-oauthlib
+google-cloud-bigquery
```
Comment (sserurich) [inline src/calibrate/requirements.txt:18]: we can specify versions for these packages

**Review by sserurich — APPROVED** [review]
worked perfectly well, except some minor issues I had with bigquery packages.. Also the error message can be updated to reflect the current parameter input. This doesn't hinder merging but recommendable 
image

File: src/calibrate/requirements.txt (hunk 1 above)
Comment (Priscilla-AL) [reply src/calibrate/requirements.txt:18]: done, thanks

**Comment by Priscilla-AL** [comment]
> worked perfectly well, except some minor issues I had with bigquery packages.. Also the error message can be updated to reflect the current parameter input. This doesn't hinder merging but recommendable
> image

sure!

**Pull request merged**